iOS/SWTableViewCell/component/component.yaml

```diff
@@ -12,6 +12,10 @@ libraries:
 samples:
   - name: "iOS Unified Sample"
     path: ../samples/SWTableViewCellSample.sln
+    installNuGets:

```

Comment:
Missing remove part of the nuget installation.  should remove the source project